trunk/source/PEAtoBoogie/src/de/uni_freiburg/informatik/ultimate/pea2boogie/req2pea/ReqCheckAnnotator.java

```diff
@@ -255,7 +319,7 @@ private Statement genAssertRTInconsistency(final Entry<PatternType<?>, PhaseEven
 		final PhaseEventAutomata[] automata = automataSet.toArray(new PhaseEventAutomata[subset.length]);
 
 		final Expression expr = mRtInconcistencyConditionGenerator.generateNonDeadlockCondition(automata);
-		final ReqCheck check = createReqCheck(Spec.RTINCONSISTENT, subset);
+		final ReqCheck check = createReqCheck(Spec.RTINCONSISTENT,"", subset);
```

Comment:
Insert a space after a parameter separator:

```suggestion
		final ReqCheck check = createReqCheck(Spec.RTINCONSISTENT, "", subset);
```